### DISCUSSION TITLE
Add the ScoreV1 mod in osu!lazer please

### DISCUSSION THREAD

**Discussion by tematikkp** [ppy/osu#20544] [Ideas] [answered]
We love the ScoreV1 in stable, please add the ScoreV1 mod in lazer (in the "Conversion" section of the mods, in the bottom of the "Classic" mod) so that everyone can enjoy the old scoring system.

**Comment by fyyrraaa** [comment, marked as answer]
Would setting "Score display mode" to "Classic" (under `Settings > Gameplay > General`) work for you?

![osu_2022-10-02_01-41-48 - crop 2022-10-02 01-43-03](https://user-images.githubusercontent.com/3055647/193443579-7c66dee7-e283-4a48-8900-dd11f714aba3.png)

**Reply by smoogipoo** [reply]
This is the intended solution.
